sphinx/index.rst

```diff
@@ -0,0 +1,10 @@
+dust C++ documentation
+======================
+
+.. toctree::
+   :maxdepth: 2
+   :caption: Contents:
+
+   random
+
+This set of pages describes the ``dust`` C++ API in more detail than that discussed in the main ``dust`` documentation.  Principally it covers the random number library.
```

Comment:
I have no idea how this whole set up works evidently and am not sure where that would go? In light of the other comments though it might not be worth getting too attached to the rst/sphinx docs